homeassistant/components/motionblinds_ble/__init__.py

```diff
@@ -0,0 +1,57 @@
+"""MotionBlinds BLE integration."""
+from __future__ import annotations
+
+import logging
+
+from motionblindsble.crypt import MotionCrypt
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.typing import ConfigType
+
+from .const import CONF_MAC_CODE, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [
+    Platform.COVER,
+]
+
+CONFIG_SCHEMA = cv.empty_config_schema(DOMAIN)
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up MotionBlinds BLE integration."""
+
+    _LOGGER.info("Setting up MotionBlinds BLE integration")
+
+    # The correct time is needed for encryption
+    _LOGGER.info("Setting timezone for encryption: %s", hass.config.time_zone)
+    MotionCrypt.set_timezone(hass.config.time_zone)
+
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up MotionBlinds BLE device from a config entry."""
+
+    _LOGGER.info("(%s) Setting up device", entry.data[CONF_MAC_CODE])
```

Comment:
please use debug logs instead

Reply:
Should I replace all info by debug, or what would you leave at info?

Reply:
debug. `info` is normally reserved for HA itself

Reply:
Thanks will replace them all

Reply:
still a bunch of infos here 😉

Reply:
I have replaced all of them, where do you still see infos?
![image](https://github.com/home-assistant/core/assets/78048721/b1f41cd6-04cd-4fed-bdec-2c40768bd415)